server/analyzer/init.go

```diff
@@ -60,7 +60,7 @@ func Init() {
 	)
 
 	// The auto-commit rule writes the contents of the context, so we need to insert our finalizer before that
-	analyzer.OnceAfterAll = insertAnalyzerRules(analyzer.OnceAfterAll, analyzer.AutocommitId, true,
+	analyzer.OnceAfterAll = insertAnalyzerRules(analyzer.OnceAfterAll, analyzer.BacktickDefaulColumnValueNamesId, false,
```

Comment:
not sure i understand, why did we change the rule

Reply:
There's no Autocommit rule anymore; it's replace with logic in `finalizeIters` now.
So this has to insert rules after the `BacktickDefaulColumnValueNamesId`